en/docs/learn/api-security/api-authentication/secure-apis-using-oauth2-tokens.md

```diff
@@ -257,3 +257,14 @@ curl -H "SERVER_AUTH: Bearer 3c536e3f-397c-3df9-a89c-9c40efedfa9e" -H "accept: a
 Note that the customized authorization header appears in the Developer Portal UI as well.
 
 [![]({{base_path}}/assets/img/learn//auth_header.png)]({{base_path}}/assets/img/learn//auth_header.png)
+
+##Sending authorization header to the backend
```

Comment:
have a space between the two hashes and the start of the word

Reply:
fixed